frontend/components/main.jsx

```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { PEOPLE } from '../helpers';
-import { xsSpacing } from '../constants';
+import { sidecar, tamarind, juniper, dustStorm, whiteLilac } from '../constants';
 import WelcomeHeader from './welcomeHeader';
 
 const tableSize = 350;
-const lockSize = 60;
-const middleLockSize = 80;
+const pictureSize = 70;
 
 const Table = styled.div`
-  position: relative;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
   width: ${tableSize}px;
-  height: ${tableSize}px;
+  padding: 5px 0;
   margin: auto;
 `;
 
-const PersonContainer = styled.div`
-  padding: ${xsSpacing}px;
+const MiddleTable = styled(Table)`
```

Comment:
no need for a separate MiddleTable defintion

Reply:
Yeah I think I justify-content: center looks the best for the middle section so I'll delete this along with all my comment changes that I was playing around with